app/models/google_drive_file.rb

```diff
@@ -0,0 +1,17 @@
+class GoogleDriveFile < ApplicationRecord
+  has_one :document, as: :documentable, dependent: :destroy
```

Comment:
Think we also need to update `delegated_type :documentable, types: %w[UploadedFile]` in `Document.rb`.

Reply:
Done!